bundle/src/bundle_meta.rs

```diff
@@ -105,12 +122,20 @@ pub struct BindingsVersionedBundle(pub VersionedBundle);
 impl BindingsVersionedBundle {
     pub fn get_v0_5_29(&self) -> Option<BundleMetaV0_5_29> {
         match &self.0 {
+            VersionedBundle::V0_6_2(bundle_meta) => Some(BundleMetaV0_5_29::from(
+                BundleMetaV0_5_34::from(bundle_meta.clone()),
+            )),
+            VersionedBundle::V0_5_34(bundle_meta) => {
+                Some(BundleMetaV0_5_29::from(bundle_meta.clone()))
+            }
             VersionedBundle::V0_5_29(bundle_meta) => Some(bundle_meta.clone()),
-            _ => None,
```

Comment:
I'm confused why this line was deleted

Reply:
It's deleted because nothing will match it. `VersionedBundle` only has 3 variants and each of them are matched here 😃